src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -57,8 +58,8 @@ export const DefinitionList = ({ work }) => {
       case 'language':
         return (
           <ul>
-            {list.map((entity, i) => (
-              <li key={`language${i.toString()}`}>{entity.language}</li>
+            {list.map((language, i) => (
```

Comment:
`forEach` here instead of `map`?

Reply:
forEach wouldn't return the render values